Specs/Petstore/generated/Swift/Sources/API.swift

```diff
@@ -15,6 +15,13 @@ public struct Petstore {
 
     /// Used to encode Dates when uses as string params
     public static let dateEncodingFormatter = DateFormatter(formatString: "yyyy-MM-dd'T'HH:mm:ssZZZZZ")
+    public static let jsonDateEncodingStrategy: JSONEncoder.DateEncodingStrategy = .formatted(Petstore.dateEncodingFormatter)
```

Comment:
Can these be made vars and each have a comment.
Also changing the date encoding strategy wouldn’t have an affect on the default encoder, so I think we can just leave it out

Reply:
PR updated. I have deleted `jsonDateEncodingStrategy` and changed `dateEncodingFormatter` to a variable. Due to this last reason, I have also changed the `defaultJSONEncoder` to a computed variable